NEWS.md

```diff
@@ -34,6 +34,8 @@
   available. (#300)
 - `connect$DELETE()` now respects the `parser` argument rather than always using
   `NULL`.
+- `get_groups()` does not attempt to paginate API requests using a search
+  prefix, as this would lead to the function hanging. (#319)
```

Comment:
Maybe a little less technical of a note here, like this? (Assuming it is correct)

```suggestion
- `get_groups()` no longer hangs when a search `prefix` is provided. (#319)
```